src/webapp/components/member-selector/MemberSelector.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useMemo } from "react";
+import styled from "styled-components";
+import { Link } from "@material-ui/core";
+
+import i18n from "../../../utils/i18n";
+import { Selector } from "../selector/Selector";
+import { AvatarCard } from "../avatar-card/AvatarCard";
+
+export type MemberOption = {
+    value: string;
+    label: string;
+    disabled?: boolean;
+    workPosition: string;
+    phone: string;
+    email: string;
+    status: string;
+    src?: string;
+    alt?: string;
+};
```

Comment:
`MemberOption` is only defined in the presentation component? The closes counterpart in the domain I can find is `TeamMember` (but fields do not exactly match). Is it used to display such entities?

Reply:
Yes,I have done this to try not to couple anything that is inside the form folder to the domain of this application, to keep this part more generic and to be able to extract it in the future. Perhaps it has been a bad idea to call it MemberSelector and it would be better to call it UserSelector that is more general